js/currencycom.js

```diff
@@ -540,7 +556,20 @@ module.exports = class currencycom extends Exchange {
                     },
                 },
                 'info': market,
-            });
+            };
+            result.push (marketObject);
+            // add for settle currencies
+            if (swap) {
+                const settleCurrencies = this.getSettlementCurrencies ('swap', 'fetchMarkets');
+                for (let i = 0; i < settleCurrencies.length; i++) {
+                    const settleCurrency = settleCurrencies[i];
+                    result.push (this.extend (marketObject, {
```

Comment:
you would need to redefined `linear` and `inverse` as well

Reply:
yeah right, i've forgot. btw, what about `ETH/BTC:USDT`?